platform/ui-next/src/components/SidePanel/SidePanel.tsx

```diff
@@ -2,38 +2,24 @@ import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import React, { useCallback, useEffect, useState } from 'react';
 import { Icons } from '../Icons';
-import { TooltipTrigger, TooltipContent, TooltipProvider, Tooltip } from '../Tooltip';
+import { TooltipTrigger, TooltipContent, Tooltip } from '../Tooltip';
 import { Separator } from '../Separator';
 
 type StyleMap = {
   open: {
-    left: { marginLeft: string };
-    right: { marginRight: string };
+    left: { marginLeft: string; marginRight: string };
```

Comment:
can you add some docs what are these, what is alignItems?

Reply:
You got it Pontiac. 👍